wandb/apis/public/runs.py

```diff
@@ -161,6 +167,64 @@ def convert_objects(self):
 
         return objs
 
+    @normalize_exceptions
+    def histories(
+        self,
+        samples: int = 500,
+        keys: list = None,
+        x_axis: str = "_step",
+        format: Literal["default", "pandas"] = "default",
```

Comment:
i think the customer who was requesting this wanted polars, let's add it here?

Reply:
correct, polars just added